Dockerfile.ma

```diff
@@ -0,0 +1,23 @@
+ARG IMAGE=alpine:3.16
+FROM ${IMAGE}
+MAINTAINER Max Neunhoeffer <max@arangodb.com>
+
+ARG TARGETARCH
+COPY bin/linux/${TARGETARCH}/arangodb /app/
```

Comment:
old Dockerfile probably could be reused with little hack we use in arangosync:
`ENV GOARCH=${TARGETARCH:-$GOARCH}`

Then use GOARCH as usual:
`COPY bin/linux/${GOARCH}/arangodb /app/`
(probably ENV could be replaced with ARG but not sure).
Otherwise LGTM :+1: